linux_pipeline/Jenkinsfile_dpdk_pipeline

```diff
@@ -169,7 +169,7 @@ if (env.BUILD_TAGS_LATEST && env.BUILD_TAGS_LATEST != "") {
         dpdkSource.add([it, 'latest'])
     }
 }
-if (env.BUILD_TAGS_STABLE && env.BUILD_TAGS_STABLE != "" && env.BUILD_TAGS_STABLE =~ dpdkVersionPattern) {
+if (env.BUILD_TAGS_STABLE && env.BUILD_TAGS_STABLE != "" && env.BUILD_TAGS_STABLE.replace("refs/tags/","") =~ dpdkVersionPattern) {
```

Comment:
You may want to remove the prefix at place where you assigned value to variable env.BUILD_TAGS_STABLE, so that in case you use this variable in some other place you no need to remove prefix. Your choice, otherwise looks ok.